tests/run_test.py

```
@@ -286,3 +286,55 @@ def test_show_remove_no_repr() -> None:
+def test_add_ignore_existing_false() -> None:
+    """Test add command when ignore_existing is False and __repr__ exists."""
+    file_name = pathlib.Path("tests/classes/existing_repr_test.py")
+    with tempfile.NamedTemporaryFile(mode="w", delete=False) as temp_file:
+        with pathlib.Path.open(file_name, mode="rt", encoding="UTF-8") as f:
+            temp_file.write(f.read())
+        temp_file_path = pathlib.Path(temp_file.name)
+
+    result = runner.invoke(crepr.app, ["add", str(temp_file_path)])
+    assert result.exit_code == 0
+    assert (
+        f"Skipping {temp_file_path}: __repr__ exists. Use --ignore-existing to override."
```

Ruff

Line too long (89 > 88)

(E501)